### PR TITLE
Map Selection

### PR DESCRIPTION
**Pull request by m4rtint**
- Everything UI Related

### CODE REVIEW

**Comment by m4rtint** [comment]
[Map Selection](https://trello.com/c/kGnO7cn6/404-map-selection)

**Comment by wbeltran777** [comment]
missingref
I'm getting a missing ref when opening up DoozyMainMenu

**Comment by m4rtint** [comment]
@wbeltran777 
As you can see in the screen shot - The forward button in the inspector is missing the references. 
Same as the back button.

If you open the children in those objects, you should see what to drag into those places. 

Technically shouldn't be missing, it's fine on my side, and they are prefabs. I'll take another look at the prefabs.

**Comment by m4rtint** [comment]
@wbeltran777 I even reimported everything, and everything works fine.

**Comment by wbeltran777** [comment]
@m4rtint sounds good, glad there's no issue, I had just pulled got an error so I thought the prefabs hadn't been set up!

File: Assets/Doozy/Engine/Resources/Data/UIButton/UIButton_Map.asset [hunk 1]
```diff
@@ -17,3 +17,4 @@ MonoBehaviour:
   Names:
   - Eight
```
Comment (Gursaanj) [inline Assets/Doozy/Engine/Resources/Data/UIButton/UIButton_Map.asset]: Maybe "Eight" should change to No name Island too for consistency

**Pull request merged**